Update business details page to use recommended UTR pattern.

russellthorn opened the pull request:
I have updated the page to follow the recommended UTR pattern - https://design.tax.service.gov.uk/hmrc-design-patterns/unique-taxpayer-reference/

Unit tests have been updated and all pass as do the following AC tests:

- business customer fe
- AWRS
- ATED
- ATED subscription
- agent client mandate

## Before
![Screenshot 2023-10-20 at 13 50 03](https://github.com/hmrc/business-customer-frontend/assets/1692222/38231395-73c0-408b-bc59-8795f1b4593f)


## After
![Screenshot 2023-10-20 at 13 49 26](https://github.com/hmrc/business-customer-frontend/assets/1692222/3fb0811d-05fb-492e-b27d-e5b3ea99516d)

russellthorn committed:
Update business details page to use recommended UTR pattern.

UTR pattern - https://design.tax.service.gov.uk/hmrc-design-patterns/unique-taxpayer-reference/

nadinJ commented on conf/messages:
```diff
@@ -88,17 +88,16 @@ bc.business-verification.radio-legend = What type of business is it?
-bc.business-verification.utr.help.answer.agent = It is issued by HMRC when you register your business or for Self Assessment. Your UTR number is made up of 10 or 13 digits. If it is 13 digits only enter the last 10.
-bc.business-verification.utr.help.answer_1 = It is issued by HMRC when you register your business or for Self Assessment. Your UTR number is made up of 10 or 13 digits. If it is 13 digits only enter the last 10.
-bc.business-verification.utr.help.answer_2 = Your accountant or tax manager would normally have your UTR.
+bc.business-verification.utr.findUTR = You can <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/find-lost-utr-number">find a lost UTR (opens in new tab)</a>
```
Could we remove the html tags from the message file please

russellthorn replied:
Hi @nadinJ I have updated this and updated the content to match the updates in the UTR pattern.
![Screenshot 2023-11-01 at 14 42 52](https://github.com/hmrc/business-customer-frontend/assets/1692222/4636aefd-9c06-4df0-acbf-2bcac78cd0a0)

paulanderson1234 replied:
This message key no longer contains html tags - resolved

russellthorn committed:
Remove html from messages file

Other work done in PR:
- content updated to match updates in HMRC UTR pattern https://design.tax.service.gov.uk/hmrc-design-patterns/unique-taxpayer-reference/
- updated the tests
- removed hint text as part the UTR pattern update

russellthorn committed:
Update h1 to be a statement and 1st input to be a label

russellthorn commented:
After agreeing with the team I have updated the h1 to be a statement and the first input label to be a question.
![Screenshot 2023-11-01 at 16 07 50](https://github.com/hmrc/business-customer-frontend/assets/1692222/23c72d2d-afd1-4107-844b-d39076f3dc48)

nadinJ commented on app/views/business_lookup_LTD.scala.html:
```diff
@@ -51,26 +51,12 @@
-    <p class="govuk-body">@messages("bc.business-verification.utr.help.answer_2")</p>
-}
-
 @helpQuestionAnswer = @{
```
This is no longer utilised and should be removed (its reference has been removed on line 70)

russellthorn replied:
These message keys have already been removed. I have just pushed another commit tidying up any other redundant message keys.

nadinJ replied:
helpQuestionAnswer is the one that needs to be removed since it was only used on line 70 and that has been removed

russellthorn replied:
Done now

nadinJ committed:
DL-9469: Fixed tests for BusinessVerificationControllerSpec

russellthorn committed:
remove space before comma in message

russellthorn committed:
Removing unused message keys

russellthorn committed:
remove unused helpQuestionAnswer
